Makefile

```diff
@@ -100,6 +100,12 @@ PYTHON         ?= python3
 GIMP           ?= $(shell command -v gimp-console)
 GIMP_FLAGS     ?= -n -i --batch-interpreter=plug-in-script-fu-eval
 
+ifneq ($(findstring GNU Image Manipulation Program version 2., $(shell $(GIMP) --version)),)
```

Comment:
Generally looks good to me, thank you @matthijskooijman - I'm wondering whether we should try to accommodate the v3 prerelease versions of GIMP here -- IIRC those were numbered v2.9*.

Reply:
Good point, but I'm not sure if we really can - I expect the changes in the gimpscript were made gradually, so there are likely 2.99 versions that need the old script, others need the new script and probably some need a mix? Also, I'd like to keep this check as simple as possible.

Reply:
Ok, yep; that seems reasonable to me :+1: